screens/ProfileScreen.js

```diff
@@ -24,12 +36,37 @@ const ProfileScreen = ({ navigation }) => {
       );
     console.log('Sign_out clicked');
   };
+
+  const updateUsername = (newUsername) => {
+    console.log('ProfileScreen received this', newUsername);
+    return auth()
+      .currentUser.updateProfile({displayName: newUsername})
+      .then((data) => {
+        console.log('confirmation from Firebase');
+        setUsername(newUsername);
+      });
+  };
+  const renderNewProfilePic = (newPicUrl) => {
+    console.log('setNewProfilePic---------',newPicUrl);
+    setProfilePicUrl(newPicUrl);
+  }
+  const getOldProfileImageRef= () => {
+    console.log('profilePicUrl---------', profilePicUrl)
+    console.log('attempt at ref--------------', profilePicUrl.split('?').shift().split('profilePics%2F').pop())
```

Comment:
Always check `if (profilePicUrl !== null)` first, otherwise this code would crash when `profilePicUrl` is `null` just because of a `console.log`